lib/active_record/json_validator/validator.rb

```diff
@@ -2,6 +2,7 @@ class JsonValidator < ActiveModel::EachValidator
   def initialize(options)
     options.reverse_merge!(message: :invalid_json)
     options.reverse_merge!(schema: nil)
+    options.reverse_merge!(validator_options: {})
```

Comment:
Why `validator_options` and not just `options`? None of the previous attributes contains `validator_`